.github/workflows/test.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
       - name: Build
         run: pnpm build
 
+      # linting of some tests depends on pretest being run so that it generates the necessary files
```

Comment:
Unnecessary note, IMO. Feel free to keep it, though.

```suggestion
```

Reply:
I wrote it so that somebody doesn't go "Well, this isn't needed because `pretest` is already run on `pnpm test`".